src/qibo/tests/test_error_mitigation_vncdr.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
```

Comment:
I think it would be simpler if you use a single file for all tests. Then you would need to do the imports and define `get_noise_model` only once. It would also be in correspondance with the code where all error mitigation methods are defined in a single file.